### DISCUSSION TITLE
Welcome to intelligent-app-workshop Discussions!

### DISCUSSION THREAD

**Discussion by thegovind** [Azure/intelligent-app-workshop#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oliverlabs** [comment]
Are you looking for contributors? I am happy to take on some feature implementations :)

**Reply by adrianwright** [reply]
Yes, happy to receive contributions @oliverlabs!
